PeapodSwipe/ProductInformationScrollView.swift

```diff
@@ -9,27 +9,34 @@
 import Foundation
 import UIKit
 
-class ProductInformationScrollView: UIScrollView {
+class ProductInformationScrollView: UIScrollView, UIScrollViewDelegate {
```

Comment:
Good catch! I totally didn't realize that I have to implement `UIScrollViewDelegate` for scrolling. Thanks! 👍